.github/workflows/sanity.yml

```diff
@@ -11,16 +11,14 @@ jobs:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
         version: [8, 11, 17]
-        impl: [hotspot]
+        impl: [temurin]
```

Comment:
The impl value should be `hotspot`.

Reply:
But according to the list here https://github.com/actions/setup-java#supported-distributions, hotspot is not a valid value. Am I missing something here?

Reply:
Do you mean that I should leave the impl set to `hotspot` and add another distribution parameter with value set to `temurin` and pass that to the action's `distribution` argument?

Reply:
Could you tag the PR as ready for review? Thanks

Reply:
Done, Thanks